OpenUserJs/OpenUserJS.org
# Some dep updates

* Delete op retested
* Check their CHANGELOGs for ones available.

---

* Delete op retested.

NOTE(S)
* Skipping *connect-mongo* for a bit since semver indicates breaking change to be thoroughly examined ... claims *mongodb*@2.x is dropped. :\
